modules/appeals_api/config/schemas/v1/10182_headers.json

```diff
@@ -93,9 +91,7 @@
     "X-VA-Claimant-Birth-Date": {
       "allOf": [
         { "description": "Non Veteran claimant's birth date" },
-        { "minLength": 10 },
-        { "maxLength": 10 },
-        { "$ref": "#/definitions/nodCreateHeadersDate" }
+        { "type": "string", "format": "date" }
```

Comment:
Turns out this file requires "allOf" because of this file: NodJsonSchemaSwaggerHelper. Fixing this up is out of scope here. Will bring up during standup and see if we even want to invest time in updating a NOD v1 related files.

Reply:
We don't. Keeping this as an `allOf` is fine :)